modules/oc-mirror-v2-about.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-oc-mirror-v2-about_{context}"]
+= About the oc-mirror plugin V2
+
+
+You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror all required {product-title} content and other images to your mirror registry by using a single tool. 
+
+oc-mirror v2 is a new implementation of oc-mirror, developed based on user feedback on the existing version. It is available as a Tech Preview feature. To use it, add the `--v2` flag to the oc-mirror command line.
```

Comment:
@subhtk let us update this as suggested in the thread here https://redhat-internal.slack.com/archives/C050P27C71S/p1717593638355159